packages/apps/workshop/stories/components/navbar.stories.js

```diff
@@ -88,15 +88,16 @@ storiesOf('Components/Navbar', module)
             placement="end"
             fixed>
           </oui-navbar-notification>
-          <oui-navbar-menu ng-switch-when="user"
+          <oui-navbar-menu
+            ng-if="asideLink.name === 'user'"
             header-breadcrumb="{{asideLink.nichandle}}"
             header-title="{{asideLink.fullName}}"
             name="{{asideLink.name}}"
             links="asideLink.subLinks"
             placement="end"
             fixed>
           </oui-navbar-menu>
-          <oui-navbar-menu ng-switch-default
+          <oui-navbar-menu ng-if="asideLink.name !== 'notifications' && asideLink.name !== 'user'"
```

Comment:
You are not using the same identation everywhere (look at https://github.com/ovh-ux/ovh-ui-kit/pull/434/files#diff-8629ae71a17eaa6715783699ca370b05R78)

Reply:
Fixed :)